docs/pages/buttons.vue

```diff
@@ -103,6 +108,57 @@
 
     </DocsPageSection>
 
+    <DocsPageSection title="Icon buttons" anchor="icon_buttons">
```

Comment:
You're getting these errors because the anchor needs to start with a `#`, e.g. `#icon_buttons`:

```
[Vue warn]: Invalid prop: custom validator check failed for prop "anchor".

found in

---> <DocsAnchorTarget>
       <DocsPageSection> at docs/common/DocsPageSection.vue
         <DocsPageTemplate> at docs/common/DocsPageTemplate/index.vue
           <Docs/pages/buttons.vue> at docs/pages/buttons.vue
             <Nuxt>
               <Default> at .nuxt/layouts/default.vue
                 <Root>
```

I'll commit changes for you